Packs/SymantecEDR/Integrations/SymantecEDR/SymantecEDR.py

```diff
@@ -0,0 +1,2367 @@
+"""
+Symantec Endpoint Detection and Response (EDR) On-Prem integration with Symantec-EDR 4.6
```

Comment:
```suggestion
Symantec Endpoint Detection and Response (EDR) On-Prem integration with Symantec-EDR. 
```

Please mention 4.6 as the version we tested this with on the README file